ArduPlane/quadplane.cpp

```diff
@@ -1763,6 +1763,10 @@ void QuadPlane::update(void)
     if (motors->armed()) {
         const bool motors_active = in_vtol_mode() || assisted_flight;
         if (motors_active && (motors->get_spool_state() != AP_Motors::SpoolState::SHUT_DOWN)) {
+            // log ANG at main loop rate
+            if (!use_fw_attitude_controllers()) {
```

Comment:
```suggestion
            if (show_vtol_view()) {
```
That is the method we use to swap the existing attitude log from FW to VTOL.